### PR TITLE
LikeManga: fix url

### LINKED ISSUE #6163: Likemanga.io to Likemanga.in url change
### Source information

Likemanga 1.4.4

### Source language

English

### Source new URL

https://likemanga.in/

### Other details

Global Update and Update in manga page seem to work, but manga list in browse, as well as page list do not. 
There is a redirect in place  ( .io --> .in).

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL has not changed yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

### 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/keiyoushi/extensions-source/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by dngonz**
Closes #6163

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/en/likemanga/src/eu/kanade/tachiyomi/extension/en/likemanga/LikeManga.kt [hunk 1]
```diff
@@ -1,303 +1,9 @@
...
-        const val URL_SEARCH_PREFIX = "slug:"
-        private val dateFormat = SimpleDateFormat("MMMM dd, yyyy", Locale.ENGLISH)
-        private val chapterPageCountRegex = Regex("""load_list_chapter\((\d+)\)""")
-    }
-}
+class LikeManga : Madara(
+    "LikeManga",
+    "https://likemanga.in",
+    "en",
+)
```
Comment (vetleledaal) [inline src/en/likemanga/src/eu/kanade/tachiyomi/extension/en/likemanga/LikeManga.kt]: Are the URLs compatible? If not you should bump `versionId`.

Comment (vetleledaal) [inline src/en/likemanga/src/eu/kanade/tachiyomi/extension/en/likemanga/LikeManga.kt]: You can add this to skip a request:
```kt
override val useNewChapterEndpoint = true
```

**Comment by AwkwardPeak7** [comment]
`https://likemanga.ink/` 🤔

**Comment by vetleledaal** [comment]
> `https://likemanga.ink/` 🤔

So, at the time of writing the current domain `.io` redirects to `.in`.
On the `.in` site it refers to `.ink` as the old site.

With that in mind I would suggest:
- Make the current extension, `.io`, use `.ink` instead. I'm guessing the theme is the same.
- Make a new extension for `.in`. This should be in another PR.

**Comment by samanta-widjaja** [comment]
Hello. This is new site from LikeManga.
You can go to [Likemanga.ink](http://likemanga.ink/) to return to the old site.
sorry for the inconvenience.

**Comment by dngonz** [comment]
Follow @vetleledaal comment, now this PR only changes the domain to `.ink`

**Review by vetleledaal — APPROVED** [review]

**Pull request merged**